dashboard/src/components/AppUpgrade/AppUpgrade.tsx

```diff
@@ -9,14 +9,14 @@ import SelectRepoForm from "../SelectRepoForm";
 import UpgradeForm from "../UpgradeForm";
 
 export interface IAppUpgradeProps {
-  app: IRelease;
+  app?: IRelease;
```

Comment:
Eek, why are these changing to optional props? (and then the bangs below saying they're always defined)

Reply:
they are undefined while the call to `getAppWithUpdateInfo` is being executed, we can avoid the bangs though